templates/BuildMyBundle/config-build.xml

```diff
@@ -79,6 +94,15 @@ ARTIFACTS_DIR=/srv/artifacts/$JOB_NAME/$BUILD_NUMBER
 mkdir -p $ARTIFACTS_DIR
 ln -s $ARTIFACTS_DIR /var/lib/jenkins/jobs/$JOB_NAME/builds/$BUILD_NUMBER/archive
 export MATRIX_OUTPUT_DIR=$ARTIFACTS_DIR
+
+# In case output scenario is not set
```

Comment:
Is this condition really needed?  The below test for `! -z $OUTPUT_SCENARIO` will be false if it's unset *or* empty. I can't see the reason for setting it to `""` in the case where it's unset.